site/docs/v1/tech/tutorials/switch-search-engines.adoc

```diff
@@ -0,0 +1,89 @@
+---
+layout: doc
+title: Switching search engines
+description: How to switch back and forth between the provided search engines
+---
+
+As of version 1.16, FusionAuth supports two search engines:
+
+* database
+* elasticsearch
+
+and you can easily switch between them. The strengths and limitations of each search engine type link:/docs/v1/tech/core-concepts/users#user-search[are documented here].
+
+This tutorial walks you through changing your search engine. Reasons for modifying it vary; you may be simplifying your deployment strategy by using the database search engine. Or perhaps you need the advanced searching options of Elasticsearch. Either way, it's an easy switch.
+
+=== Prerequisites
+
+This tutorial assumes you have both the application and the search services running on Linux. You also must be running at least version 1.16.0. 
+
+If you don't, head on over to the link:/docs/v1/tech/installation-guide/fusionauth-search[installation guide]. If you already have the application running, but need to install the search service, link:/docs/v1/tech/installation-guide/fusionauth-search[here are instructions].
+
+If you are running FusionAuth via Docker, a Windows service, or some other mechanism, you'll have to stop and start FusionAuth in an installation specific manner.
+
+=== From Elasticsearch to Database
+
+Stop FusionAuth:
+
+```
+sudo service fusionauth-app stop
+sudo service fusionauth-search stop
```

Comment:
This assumes I'm running Elasticsearch on the same box, and with the provided FusionAuth search bundle. It could be a remote Elasticsearch service, local docker, etc. 

In reality if we are moving away from Elasticsearch, we don't need to stop it. The main reason to stop it would - if we are running it locally or for the purpose of FusionAuth alone, then it is no longer needed.

Reply:
Hmm. If they are running elasticsearch separately, they probably aren't managing it via the `fusionauth-search` service. This whole start/stop thing really depends on how you install FA. As I think about it, it probably makes sense to omit the actual command and rely on them to know how to do this.

Reply:
Yes, I think that is fine.